apps/studio/src/server/modules/folder/folder.router.ts

```diff
@@ -26,7 +26,7 @@ export const folderRouter = router({
         .where("parentId", "=", String(input.resourceId))
         .execute()
       const children = childrenResult.map((c) => {
-        if (c.draftBlobId || c.mainBlobId) {
+        if (c.draftBlobId || c.versionId) {
```

Comment:
issue: I think this should be `publishedVersionId`?

```suggestion
        if (c.draftBlobId || c.publishedVersionId) {
```